pkgs/applications/video/olive-editor/default.nix

```diff
@@ -1,34 +1,67 @@
-{ lib, stdenv, fetchFromGitHub
-, pkg-config, which, qmake, wrapQtAppsHook
-, qtmultimedia, frei0r, opencolorio_1, ffmpeg-full, CoreFoundation }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, which
+, frei0r
+, opencolorio
+, ffmpeg-full
+, CoreFoundation
+, cmake
+, openimageio2
+, openexr_3
+, portaudio
+, imath
+, qt6
```

Comment:
I'm not that into qt packaging, but perhaps indeed using `qt6Packages.callPackage` in `top-level/all-packages.nix` might be neater? (I think @Tungsten842 suggested that before but I can't find that comment anymore?)

Reply:
Not for this PR, definitely another PR to change the way this package is called.

Reply:
This PR updates olive from Qt5 to Qt6, but doesn't change that `all-packages.nix` uses `libsForQt5`, that seems inconsistent... how about https://github.com/MatthewCroughan/nixpkgs/pull/1 ?

Reply:
Oh okay, that's right. I didn't realise it was already being callPackaged in that way. I'm really surprised that qt5's callPackage works despite me using qt6 everywhere.